game/static/game/css/board.css

```
@@ -1950,6 +1977,94 @@ body.blindfold-mode .capture-ring {
     border-radius: 1px;
 }
 
+/* =======================================================
+                        Hint Highlight 
+==========================================================*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)